life_expectancy/tests/test_main.py

```diff
@@ -0,0 +1,10 @@
+"""Tests for the main module"""
+import pandas as pd
+from life_expectancy.main import main
+
+def test_main(pt_life_expectancy_expected):
```

Comment:
Use patch or monkeypatch so that main also consumes your fixtures and not the actual data in the `data` directory.